app/react/Layout/DocumentCounter.tsx

```diff
@@ -6,10 +6,17 @@ interface EntityCounterProps {
   entityListCount: number;
   entityTotal: number;
   totalConnectionsCount: number;
+  maxedTotalEntities: string;
 }
 
 export const DocumentCounter = (props: EntityCounterProps) => {
-  const { totalConnectionsCount, selectedEntitiesCount, entityListCount, entityTotal } = props;
+  const {
+    totalConnectionsCount,
+    selectedEntitiesCount,
+    entityListCount,
+    entityTotal,
+    maxedTotalEntities,
```

Comment:
I am unsure about the semantics of this variable:  the values could be 'eq' or 'gte'.  So, `maxedTotalEntities greater than` is not really correct.  Should we keep the original naming and go with something like `hitsTotalRelation`?  Something else?

Reply:
Yes, I struggled to come up with a reference that made semantic sense. Lets go with hitsTotalRelation